tests/functionaltests/Tests/UIKitTests/UIButtonTests.mm

```diff
@@ -118,68 +117,1128 @@ - (void)dealloc {
         ASSERT_EQ_MSG(xamlElement.Get(), nullptr, "Unexpectedly able to reacquire a strong reference to the backing Xaml element.");
     }
 
-    TEST_METHOD(UIButton_TitleColorChanged) {
-        __block auto uxEvent = UXEvent::CreateManual();
+    TEST_METHOD(UIButton_CreateButtonWithDefaultInit) {
+        StrongId<UIButtonWithControlsViewController> buttonVC;
+        buttonVC.attach([[UIButtonWithControlsViewController alloc] init]);
+        UXTestAPI::ViewControllerPresenter testHelper(buttonVC);
+
+        UIButton* buttonToTest = [buttonVC defaultButton];
+
+        WXCControl* xamlControl = rt_dynamic_cast([WXCControl class], [buttonToTest xamlElement]);
+        ASSERT_OBJCNE(xamlControl, nil);
+
+        // Extract UIButton.titleLabel control to verify its visual state
+        WXFrameworkElement* titleLabelXamlElement = [buttonToTest.titleLabel xamlElement];
+        ASSERT_OBJCNE(titleLabelXamlElement, nil);
+
+        WXCTextBlock* textBlock = rt_dynamic_cast([WXCTextBlock class], titleLabelXamlElement);
+        ASSERT_OBJCNE(textBlock, nil);
+
+        // Background color
+        dispatch_sync(dispatch_get_main_queue(), ^{
+            WUXMSolidColorBrush* solidBackgroundBrush = rt_dynamic_cast([WUXMSolidColorBrush class], xamlControl.background);
+            UIColor* expectedBackgroundColor = [UIColor colorWithRed:1.0f green:1.0f blue:1.0f alpha:0.0f];
+            EXPECT_TRUE(UXTestAPI::IsRGBAEqual(solidBackgroundBrush, expectedBackgroundColor));
+        });
+
+        // Title color
+        dispatch_sync(dispatch_get_main_queue(), ^{
+            WUXMSolidColorBrush* solidForegroundBrush = rt_dynamic_cast([WUXMSolidColorBrush class], textBlock.foreground);
+
+            // BUGBUG: Thought the default should be whiteColor
+            UIColor* expectedTitleColor = [UIColor blackColor];
```

Comment:
This test will be updated shortly - the title color is being updated to a white color but the timing of this test picks up the XAML default.